nipype/pipeline/engine/utils.py

```diff
@@ -1021,7 +1022,10 @@ def export_graph(graph_in, base_dir=None, show=False, use_execgraph=False,
                                suffix='.dot',
                                use_ext=False,
                                newpath=base_dir)
-    nx.write_dot(pklgraph, outfname)
+    if LooseVersion(nx.__version__) < LooseVersion('1.11'):                           
+        nx.write_dot(pklgraph, outfname)
+    else:
+        nx.drawing.nx_pydot.write_dot(pklgraph, outfname)
```

Comment:
@mwaskom - it's possible that they changed the api here completely.

@polosecki - could you please check if a former version of nx has this function, in which case you don't need the `if ... else ...``` clause.